keps/sig-api-machinery/1281-network-proxy/README.md

```diff
@@ -95,15 +95,15 @@ Implies no overlapping (i.e. shared) IPs on the network.
 
 ## Proposal
 
-We will run a connectivity server inside the master network.
+We will run a connectivity server inside the control plane network.
 It could work on either a HTTP Connect mechanism or gRPC.
 For the alpha version we will attempt to get this working with HTTP Connect.
 We will evaluate HTTP Connect for scalability, error handling and traffic types.
 For scalability we will be looking at the number of required open connections.
 Increasing usage of webhooks means we need better than 1 request per connection (multiplexing).
 We also need the tunnel to be tolerant of errors in the requests it is transporting.
 HTTP-Connect only supports HTTP requests and not things like DNS requests.
-We assume that for HTTP URL request,s it will be the proxy which does the DNS lookup.
+We assume that for HTTP URL requests, it will be the proxy which does the DNS lookup.
```

Comment:
request(s)

Reply:
This PR is merged, I will pick this up in my next pass.